Source/RCellFinder_Zombiefied.cs

```diff
@@ -1168,14 +1141,13 @@ private static bool TryFindSiegePosition(IntVec3 entrySpot, float minDistToColon
                                     if (!randomCell.Roofed(map))
                                     {
                                         int num3 = 0;
-                                        CellRect.CellRectIterator iterator = CellRect.CenteredOn(randomCell, 10).ClipInsideMap(map).GetIterator();
-                                        while (!iterator.Done())
+                                        foreach (IntVec3 cell in CellRect.CenteredOn(randomCell, 10).ClipInsideMap(map))
                                         {
+                                            // Should this use the iterated variable, "cell", rather than "randomCell"?
                                             if (randomCell.SupportsStructureType(map, TerrainAffordanceDefOf.Heavy) && randomCell.SupportsStructureType(map, TerrainAffordanceDefOf.Light))
```

Comment:
Pretty curious on this line, seems like it just adds up the count, but Is that what we want to do here?